db/migrate/20150617021911_create_course_lesson_plan_milestones.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCourseLessonPlanMilestones < ActiveRecord::Migration
+  def change
+    create_table :course_lesson_plan_milestones do |t|
+      t.string :title, null: false
+      t.text :description
+      t.timestamp :start_time, null: false
+
+      t.timestamps null: false
```

Comment:
Sorry, in this case the milestone is not a lesson plan item so you do need userstamps.